sdk/security/keyvault/azkeys/CHANGELOG.md

```diff
@@ -3,9 +3,11 @@
 ## 0.10.0 (Unreleased)
 
 ### Features Added
+* Upgraded to API version 7.4
 
 ### Breaking Changes
 * Moved from `sdk/keyvault/azkeys` to `sdk/security/keyvault/azkeys`
+* Renamed `ActionType` to `KeyRotationPolicyAction`
```

Comment:
We'll release the old module first, so this won't be a breaking change and this module won't have new features. The only change between `sdk/keyvault/azkeys` and `sdk/security/keyvault/azkeys` will be the import path.

Reply:
fixed in [4f8258d](https://github.com/Azure/azure-sdk-for-go/pull/20499/commits/4f8258da818f1693219bd974f2bcef471252c69b)